tf-modules/archive/api.tf

```diff
@@ -10,6 +10,18 @@ resource "aws_cloudwatch_log_group" "api" {
   tags              = local.default_tags
 }
 
+resource "aws_secretsmanager_secret" "api_cmr_password" {
+  name_prefix = "${var.prefix}-api-cmr-password"
+  description = "CMR password for the Cumulus API's ${var.prefix} deployment"
+  tags        = local.default_tags
+}
+
+resource "aws_secretsmanager_secret_version" "api_cmr_password" {
+  count         = length(var.cmr_password) == 0 ? 0 : 1
```

Comment:
We need to update the `default` for `cmr_password` variable in the archive module to an empty string:

https://github.com/nasa/cumulus/blob/master/tf-modules/archive/variables.tf#L15

Reply:
Good catch.